src/redux/actions/samples/loadSamples.js

```diff
@@ -1,22 +1,79 @@
 import fetchAPI from 'utils/http/fetchAPI';
 import handleError from 'utils/http/handleError';
+import config from 'config';
+import { api } from 'utils/constants';
 
 import {
   SAMPLES_LOADED,
   SAMPLES_ERROR,
   SAMPLES_LOADING,
 } from 'redux/actionTypes/samples';
 
+const toApiV2 = (samples) => {
+  const apiV2Samples = [{ samples: {} }];
+
+  const buildApiv1FilesObject = (fileObject) => {
+    const fileNames = [];
+    const apiV1FileObject = {};
+
+    Object.keys(fileObject).forEach((key) => {
+      const fileName = fileObject[key].s3Path.split('/').pop();
+      fileNames.push(fileName);
+
+      apiV1FileObject[fileName] = {
+        path: fileObject[key].s3Path,
+        size: fileObject[key].size,
+        valid: fileObject[key].valid,
+        name: fileName,
+        upload: {
+          status: fileObject[key].uploadStatus,
+        },
+      };
+    });
+    return { apiV1FileObject, fileNames };
+  };
+
+  const sampleTechnologyConvert = (technology) => {
+    switch (technology) {
+      case '10x': return '10X Chromium';
+      default: return technology;
+    }
+  };
+
+  samples.data.message.forEach((sample) => {
+    const { apiV1FileObject, fileNames } = buildApiv1FilesObject(sample.files);
+    apiV2Samples[0].samples[sample.id] = {
```

Comment:
Why do we store this in `[0]`?

Reply:
because that is how the apiv1 object looks like and i wanted to mimick it so its exactly the same format